### DISCUSSION TITLE
[CI] Run only needed job

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#7158] [Ideas]
Hi, 
i have a suggest for optimize CI
For example if commit or PR affect: 
- only doc, then run build/ci/.azure-devops-docs.yml 
- only vsix then run build/ci/.azure-devops-docs.yml 
- ecc ...

**Comment by jeromelaban** [comment]
Thanks for the suggestion. This needs adjustments that are needed on the azure devops side (a work in progress) that cannot be done at this point.

**Reply by workgroupengineering** [reply]
I understand.
You could inspect the path generated by the command `git log -m -1 --name-only --pretty="format:"`
if contain only path ./doc/ run build/ci/.azure-devops-docs.yml
ecc ...
